source/sass/blocks/goods.scss

```
@@ -0,0 +1,19 @@
+.goods--title {
```

И тут надо незабыть поправить.
```suggestion
.goods__title {
```